internal/manifest/readycheck/ready_check.go

```diff
@@ -1,14 +1,10 @@
-package manifest
+package readycheck
```

Comment:
Regarding the second point about scheme, I suggest postponing this till we do the Statefulset state determination because we are not yet sure that we won't need the client to list the pods.